doc/source/learning/airship.rst

```diff
@@ -0,0 +1,181 @@
+Airship
+=======
+
+Airship is a platform that takes YAML documents and turns them into Clouds.
+Takes helm charts, baremetal, security policies and network description and
+creates a cloud
+
+.. image:: https://airship-treasuremap.readthedocs.io/en/stable/_images/architecture.png
+
+* Airship architecture
+    https://airship-treasuremap.readthedocs.io/en/stable/ (Stable)
+
+    https://airship-treasuremap.readthedocs.io/en/latest/ (Latest)
+
+* Git repository
+    https://opendev.org/airship
+
+* Video
+    https://www.youtube.com/watch?v=0eEisMm9ykg
+
+
+Airship architecture parts
+--------------------------
+
+Shipyard
+++++++++
+
+Shipyard is the cluster lifecycle orchestrator that provides to end-users the
+framework to orchestrate and deploy a fully functional container-based Cloud.
+
+* Documentation
+    https://airship-shipyard.readthedocs.io/
+
+* Git repository
+    https://opendev.org/airship/shipyard
+
+
+Treasuremap
++++++++++++
+
+This documentation project outlines a reference architecture for automated
+cloud provisioning and management, leveraging a collection of interoperable
+open-source tools.
+
+* Documentation
+    https://airship-treasuremap.readthedocs.io/en/latest/index.html
+
+* Git repository
+    https://opendev.org/airship/treasuremap
+
+Armada
+++++++
+
+Armada uses a YAML to describe a multi-chart deployment. The keys are:
+
+* Armada communicates with helm to create the desired infrastructure.
+* The configuration can be stored on the Deckhand.
+* Armada is commanded by Shipyard
+
+* Documentation
+    https://airship-armada.readthedocs.io/
+
+* Git repository
+    https://opendev.org/airship/armada
+
+* Guide
+    https://airship-armada.readthedocs.io/en/latest/operations/guide-use-armada.html
+
+* Video
+    https://www.youtube.com/watch?v=ae-g9irTsXY
+
+
+Deckhand
+++++++++
+
+Deckhand is a document-based configuration storage service using Barbican
+
+* Documentation
+    https://airship-deckhand.readthedocs.io/en/latest/
+
+* Git repository
+    https://opendev.org/airship/deckhand
+
+
+Pegleg
+++++++
+
+Pegleg is a document aggregator that combine all the required documents across
+multiple Git repositories, additionally checks the documents with lint and
+validations via Deckhand.
+
+* Documentation
+    https://airship-pegleg.readthedocs.io/
+
+* Git repository
+    https://opendev.org/airship/pegleg
+
+
+
+Airship setup process
```

Comment:
This whole section could be removed. We have the process as part of the deployment guide, which devs should read first.

Reply:
Section removed.